src/coreclr/vm/methodtablebuilder.cpp

```diff
@@ -5023,15 +5020,11 @@ MethodTableBuilder::ValidateMethods()
         {
             if (!IsMdVirtual(it.Attrs()))
             {   // Non-virtual methods cannot participate in a methodImpl pair.
+                /* TODO: we should probably check that we're overriding a virtual static method here
                 BuildMethodTableThrowException(IDS_CLASSLOAD_MI_MUSTBEVIRTUAL, it.Token());
+                */
             }
         }
-
```

Comment:
We should not permit static virtual methods on classes/structs. By our ecma design, those should continue to throw.